extensions/wikia/Forum/DiscussionsHelper.class.php

```diff
@@ -0,0 +1,50 @@
+<?
+
+class DiscussionsHelper {
+
+	/**
+	 * @return bool
+	 */
+	public static function areForumsArchivedAndDiscussionsEnabled() {
+		return true;
+//		global $wgArchiveWikiForums, $wgEnableDiscussions;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)